flyway/postgre/migrations/sql/V1_3__Create_lesson2_tables.sql

```diff
@@ -1,69 +1,86 @@
-CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+DROP TYPE IF EXISTS organization_status_type;
+DROP TYPE IF EXISTS user_status_type;
+DROP TYPE IF EXISTS sex_type;
 
 CREATE TYPE organization_status_type AS ENUM ('active', 'suspend', 'deleted');
 CREATE TYPE user_status_type AS ENUM ('active', 'deactivated', 'deleted');
-CREATE TYPE gender_type AS ENUM ('male', 'female');
+CREATE TYPE sex_type AS ENUM ('male', 'female');
+
+DROP TABLE IF EXISTS sso_organization;
+DROP TABLE IF EXISTS sso_user;
+DROP TABLE IF EXISTS sso_roles;
+DROP TABLE IF EXISTS sso_user_profile;
+DROP TABLE IF EXISTS sso_organization_user;
+DROP TABLE IF EXISTS sso_user_role;
+
+
 
 CREATE TABLE IF NOT EXISTS sso_organization (
-    organization_id SERIAL PRIMARY KEY NOT NULL,
-    organization_name VARCHAR(255) NOT NULL,
-    status organization_status_type DEFAULT 'active'::organization_status_type NOT NULL,
+    organization_id INT NOT NULL,
+    organization_name VARCHAR(100) NOT NUll,
+    kvp JSON,
+    date_created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    date_modified TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    status organization_status_type DEFAULT 'active',
     address VARCHAR(255),
-    contact_number VARCHAR(255),
-    email VARCHAR(255),
+    contact_number VARCHAR(100),
+    email VARCHAR(100),
     country VARCHAR(255),
     state VARCHAR(255),
     city VARCHAR(255),
-    postal_code VARCHAR(255),
-    website_url VARCHAR(255),
-    kvp JSONB,
-    date_created TIMESTAMPTZ DEFAULT now(),
-    date_modified TIMESTAMPTZ DEFAULT now()
+    postal_code VARCHAR(10),
+    organization_sub_name VARCHAR(10),
+    website_url VARCHAR(255)
+    PRIMARY KEY (organization_id)
 );
 
-CREATE TABLE IF NOT EXISTS sso_role (
-    role_id UUID DEFAULT uuid_generate_v4() PRIMARY KEY NOT NULL,
-    role_name VARCHAR(255) NOT NULL,
-    role_description VARCHAR(255),
-    organization_id INT,
-    FOREIGN KEY (organization_id) REFERENCES sso_organization(organization_id)
+CREATE TABLE IF NOT EXISTS sso_user (
+    user_id UUID DEFAULT uuid_generate_v4(),
```

Comment:
Khóa chính thì mình nên không cho phép NULL

```suggestion
    user_id UUID DEFAULT uuid_generate_v4() NOT NULL,
```